tools/smoke-tests/validation-tests/src/test/java/io.spine/validate/ValidatingBuilderTest.java

```diff
@@ -175,91 +180,55 @@ void dispense_with_unchecked_map_fields() {
     }
 
     @Test
-    @DisplayName("not allow to mutate a (set_once) marked string field via the setter method")
-    void testSetDoesNotAllowMutate() {
-        TaskVBuilder builder = TaskVBuilder
-                .newBuilder()
-                .setId(Identifier.newUuid());
-        assertThrows(ValidationException.class, () -> builder.setId(Identifier.newUuid()));
+    @DisplayName("not allow to change the value of a (set_once) field")
+    void testSetOnceFieldOption() {
+        Stream<Function<TaskVBuilder, ?>> stringFieldMutations = Stream.of(
+                taskVBuilder -> taskVBuilder.setId(newUuid()),
+                taskVBuilder -> taskVBuilder.mergeFrom(sampleTask()),
+                TaskVBuilder::clearId
+        );
+        Stream<Function<TaskVBuilder, ?>> enumFieldMutations = Stream.of(
+                taskVBuilder -> taskVBuilder.setLabel(CRITICAL),
+                taskVBuilder -> taskVBuilder.mergeFrom(sampleTask()),
+                TaskVBuilder::clearLabel
+        );
+        Stream<Function<TaskVBuilder, ?>> messageFieldMutations = Stream.of(
+                taskVBuilder -> taskVBuilder.setAssignee(Member.getDefaultInstance()),
+                taskVBuilder -> taskVBuilder.mergeFrom(sampleTask()),
+                TaskVBuilder::clearAssignee
+        );
+
+        testFields(taskVBuilder -> taskVBuilder.setId(newUuid()), stringFieldMutations);
+        testFields(taskVBuilder -> taskVBuilder.setLabel(IMPORTANT), enumFieldMutations);
+        testFields(taskVBuilder -> taskVBuilder.setAssignee(member()), messageFieldMutations);
```

Comment:
Please split this test case into many. If we perform multiple similar checks, then there are multiple reasons for the test to fail.